install.sh

```diff
@@ -215,6 +215,17 @@ if [[ $(uname) == "Darwin" ]]; then
   fi
 fi
 
+if [[ $(hostname) == tioga* ]]; then
+  if [[ ! -f $CONDA_PREFIX/etc/OpenCL/vendors/amd_ceesd.icd ]]; then
+      echo "**** Installing AMD OpenCL ICD (rocm) for Tioga"
+    if [[ -n $ROCM_PATH ]]; then
+      echo "**** Warning: No rocm module loaded, assuming v6.1.2"
+      ROCM_PATH="/opt/rocm-6.1.2"
+    fi
+    echo "$ROCM_PATH/lib/libamdocl64.so" > $CONDA_PREFIX/etc/OpenCL/vendors/amd_ceesd.icd
```

Comment:
Unfortunately, there is no way to use env variables in the actual icd file...

Reply:
What about just failing if no rocm module is loaded?  That way avoid sticking the wrong one...